### PR TITLE
feat: add support border color setting

### PR DESCRIPTION
**Pull request by Flunt1k**

### CODE REVIEW

File: src/plugins/highcharts/renderer/helpers/config/options.js [hunk 1]
```diff
@@ -216,15 +216,7 @@ const options = {
             statesForLine,
         ),
         areaspline: first,
-        bar: Object.assign(
-            {
-                borderWidth: 0,
-                pointWidth: 4,
```
Comment (Flunt1k) [inline src/plugins/highcharts/renderer/helpers/config/options.js:222]: remove this props due to it always overwrites the default width of bars, thereby forcing user to set pointWidth value null (which is incorrect) or the fixed value.

File: src/plugins/highcharts/__stories__/Bar.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+import {ChartKit} from '../../../components/ChartKit';
+import {ChartKitRef} from '../../../types';
+import {settings} from '../../../libs';
+import {HighchartsPlugin} from '../index';
+import {defaultChartKitPropsControlsState} from './constants/story-settings';
+import holidays from './mocks/holidays';
+import {data} from './mocks/bar';
+
+export default {
+    title: 'Plugins/Highcharts/Column',
```
Comment (korvin89) [inline src/plugins/highcharts/__stories__/Bar.stories.tsx:13]: `.../Bar` ?

File: src/plugins/highcharts/__stories__/Bar.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+    return (
+        <ThemeProvider theme={args.theme}>
+            <div style={{height: '600px', width: '100%'}}>
+                <ChartKit
+                    ref={chartkitRef}
+                    id="1"
+                    type="highcharts"
+                    data={data}
+                    hoistConfigError={false}
```
Comment (korvin89) [inline src/plugins/highcharts/__stories__/Bar.stories.tsx:44]: Redundant property. It makes sense only in case of empty data or huge amount of series

**Review by korvin89 — APPROVED** [review]

File: src/plugins/highcharts/__stories__/Bar.stories.tsx (hunk 2 above)
Comment (korvin89) [reply src/plugins/highcharts/__stories__/Bar.stories.tsx:13]: I got it)

**Pull request merged**